builder_data.yaml

```diff
@@ -61,7 +61,7 @@ apt_repo_command: |
   ) 200>/var/lock/apt-repo-lock
 nonfinal_apt_repo_command: |
   (
-    flock --wait 600 200
+    flock --wait 600 400
```

Comment:
I'm assuming this is probably okay, but how do we know that 400 is a valid/open file descriptor?

Reply:
Based on some Googling, it looks like there's a soft limit of 1024. 
File descriptors get assigned in order - there are some standard ones, like stdin, but when I first wrote this, the source I used seemed to confidently say that high numbers would be fine. I originally used 200 based on that example, so 400 should be fine as well.